copier.yaml

```diff
@@ -11,7 +11,8 @@ _envops:
   variable_start_string: "{{"
 
 _tasks:
-  - '(cd apps/{{api}}; npx npm-add-script -f -k bootstrap -v "talos put --api \${{api|upper}}_BASE --endpoint \$SPARQL_ENDPOINT --user \$SPARQL_USER --password \$SPARQL_PASSWORD")'
+  - "cd apps && cd {{api}} && python3 _make_npm_script.py || python _make_npm_script.py"
```

Comment:
_(from discussion in another place)_ 
@tpluscode suggests that if we are forced to do this logic in scripts anyway, let's just do it in .JS "natively" instead of this "python file->in-python shell->in-shell npm" approach here.
(Will require some extra work with npm dependencies so let's not do it in this PR) 
Example:  https://github.com/coleww/npm-add-script#api